### PR TITLE
Update cluster-proxy-addon chart to fix a security issue.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

The related issue: https://github.com/stolostron/backlog/issues/26546
The related fix PR: https://github.com/stolostron/cluster-proxy-addon/pull/81

### CODE REVIEW

File: pkg/templates/charts/toggle/cluster-proxy-addon/templates/user-deployment.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     component: cluster-proxy-addon-user
     chart: cluster-proxy-addon-2.1.0
 spec:
-  # fixed replicas to 1, the kubectl-ui does not support multiple replicas
+  # fixed replicas to 1, the user-server does not support multiple replicas
```
Comment (JakobGray) [inline pkg/templates/charts/toggle/cluster-proxy-addon/templates/user-deployment.yaml:10]: This caught my eye. This says the deployment must be fixed to 1 replica yet below the replica count is configurable.  @xuezhaojun will this cause problems?

Comment (xuezhaojun) [reply pkg/templates/charts/toggle/cluster-proxy-addon/templates/user-deployment.yaml:10]: It won't, I think it's a outdate comments, I will fix later.

**Pull request merged**